webapp/src/js/controllers/inbox.js

```diff
@@ -470,6 +471,7 @@ var feedback = require('../modules/feedback'),
       Language()
         .then(function(language) {
           moment.locale([language, 'en']);
+          SetDatepickerLanguage(language);
           $translate.use(language);
```

Comment:
Do you think it would make sense to just call the `SetLanguage` service here instead of duplicating the code?